### PR TITLE
chore(bundlemon): run Bundlemon everywhere + make it sensitive

### PR DESCRIPTION
**Pull request by trollepierre**
- [ ] Changelog updated if needed

My goal is to set the maxPercentIncrease according to what is needed. Maybe some PR will fail and need to update this.

### CODE REVIEW

**Comment by Crash--** [comment]
Oh nice!

File: .bundlemonrc [hunk 1]
```diff
@@ -1,46 +1,235 @@
...
+      "path": "drive/app/drive.<hash>.min.css",
       "maxSize": "55 KB"
     },
     {
-      "path": "public/cozy-bar.css",
+      "path": "drive/img/app-icon.<hash>.svg",
+      "maxSize": "1 KB"
+    },
+    {
+      "path": "drive/intent/drive.<hash>.js",
```
Comment (trollepierre) [inline .bundlemonrc:17]: typo x3

File: .bundlemonrc [hunk 2]
```diff
@@ -1,46 +1,235 @@
...
+    {
+      "path": "drive/public-drive.<hash>.min.css",
+      "maxSize": "86 KB"
+    },
+    {
+      "path": "drive/vendors-drive.<hash>.min.css",
+      "maxSize": "1 B"
+    },
+    {
+      "path": "photos/app/photos.<hash>.js",
```
[the remark this answers is not in the document]
Comment (Crash--) [reply .bundlemonrc:113]: Because we only run bundlemon on Drive: https://github.com/cozy/cozy-drive/blob/master/.travis.yml#L56

**Comment by Crash--** [comment]
Why there is no /photos/ files in the report?

**Comment by trollepierre** [comment]
I retry on https://github.com/cozy/cozy-drive/pull/2669

We are launching report 2 times in raw (once for Drive, then Photos)
I believe it's not easy for Bundlemon to update the report once it's generated.

If we really want one full report, we need to use yarn build:drive && yarn build:photos && yarn bundlemon 
this would remove one job in the CI. Should we try that?

**Comment by trollepierre** [comment]
#2669 useless compared to this one.

**Comment by Crash--** [comment]
> If we really want one full report, we need to use yarn build:drive && yarn build:photos && yarn bundlemon
this would remove one job in the CI. Should we try that?

Why not removing yarn bundlemon from the `build-web` alias and add a new job at the end that run bundlemon?

**Comment by trollepierre** [comment]
> Why not removing yarn bundlemon from the build-web alias and add a new job at the end that run bundlemon?

Do you have a quick solution to set build from Drive and build from Photos in the cache, to reuse it in the next step?

**Comment by Crash--** [comment]
Cache is not shared between stage?

**Comment by trollepierre** [comment]
> Cache is not shared between stage?

No, I do not think we do that in travis  https://app.travis-ci.com/github/cozy/cozy-drive/jobs/580480498
I guess that is why Cozy-UI uses only one long job (for build + argos).

ps: this is possible to do with CircleCI. [Example](https://github.com/trollepierre/recontact_travel_blog/blob/dev/.circleci/config.yml#L31).
I will ask if anyone knows how to handle that.

**Comment by trollepierre** [comment]
Idea: using     "build": "yarn build:drive & yarn build:photos", <= to parallelize the job.
then run bundlemon (in the same job)

**Comment by trollepierre** [comment]
@Crash-- fixed without photos

**Comment by Crash--** [comment]
conflicts to solve

**Pull request merged**